scopes/explorer/ui/base-component-card/details/details.module.scss

```diff
@@ -28,6 +28,15 @@
     margin-right: 5px;
   }
 }
+.versionHolder {
+  display: flex;
+  align-items: center;
+  overflow: hidden;
+  > img {
```

Comment:
you pass an img class but are using this css on the img element. please use the class

Reply:
my mistake, thanks.